Week4/app/src/main/java/com/kulloveth/moviesapp/db/room/MovieDatabse.kt

```diff
@@ -46,9 +47,8 @@ abstract class MovieDatabse : RoomDatabase() {
         ).addCallback(object : RoomDatabase.Callback() {
             override fun onCreate(db: SupportSQLiteDatabase) {
                 super.onCreate(db)
-                GlobalScope.launch(context = Dispatchers.IO) {
-                    Injection.provideRepository.insertAllMovie(
-                        MoviesDataManager.movieList)
+                CoroutineScope(context = Dispatchers.IO).launch {
+                    Injection.provideRepository.insertAllMovie(MoviesDataManager.movieList)
                 }
```

Comment:
Nice job removing allow main thread queries.

Reply:
OH looking at this I actually removed the global scope before I submitted, kindly take a look